.travis.yml

```diff
@@ -20,6 +20,7 @@ matrix:
       env: NOX_SESSION=test-3.7
     - python: 3.8-dev
       env: NOX_SESSION=test-3.8
+      dist: bionic
```

Comment:
Also, can we just mention here that we’re mostly setting this to get OpenSSL 1.1.1+?

Reply:
Will add a comment